### ISSUE TITLE
JAWS: was unable to complete protocol, most likely due to one of the issues identified from the other SR results

### ISSUE THREAD

**Issue by yannisbarlas** [Coko-Foundation/hhmi#342]
> **Migrated from GitLab**
> **Original Author:** Yannis Barlas (`@yannis`)
> **Original Link:** https://gitlab.coko.foundation/coko-org/products/hhmi/hhmi/-/issues/139

No description provided.
